kinto_wizard/validate.py

```diff
@@ -0,0 +1,71 @@
+try:
+    from jsonschema import Draft7Validator as SchemaValidator
+except ImportError:  # pragma: no cover
+    from jsonschema import Draft4Validator as SchemaValidator
```

Comment:
Is this really necessary since we have `--pre` everywhere?

Reply:
Done with #63